test/unit-test/cellular_pkthandler_utest.c

```diff
@@ -620,6 +640,29 @@ void test__Cellular_HandlePacket_Wrong_RespType( void )
     TEST_ASSERT_EQUAL( CELLULAR_PKT_STATUS_BAD_PARAM, pktStatus );
 }
 
+/**
+ * @brief Test that URC with colon for _Cellular_HandlePacket when token is not in the token table.
+ */
+void test__Cellular_HandlePacket_AT_UNSOLICITED_Input_String_With_Colon_Not_In_Urc_Token_Path( void )
+{
+    CellularContext_t context;
+    CellularPktStatus_t pktStatus = CELLULAR_PKT_STATUS_OK;
+
+    memset( &context, 0, sizeof( CellularContext_t ) );
+    /* copy the token table. */
+    ( void ) memcpy( &context.tokenTable, &tokenTableWoParseFunc, sizeof( CellularTokenTable_t ) );
+    Cellular_ATStrDup_StubWithCallback( _CMOCK_Cellular_ATStrDup_CALLBACK );
```

Comment:
This stub should be used in the other cases that test URC handling.

Reply:
In other words, anywhere that currently has _Cellular_GenericCallback_Ignore, should instead set the same Cellular_ATStrDup_StubWithCallback callback and verify the result.